test/cmd/newapp.sh

```diff
@@ -81,9 +81,23 @@ oc delete template ruby-helloworld-sample
 oc new-app https://github.com/openshift/ruby-hello-world -l app=ruby
 oc delete all -l app=ruby
 
-# allow use of non-existent image
+if `hash docker > /dev/null 2>&1`; then
```

Comment:
This really shouldn't be here and should be in an integration test.  test/cmd should *not* prereq Docker.  Please open a follow up.